src/modules/home/types/accommodation.type.ts

```diff
@@ -1,8 +1,10 @@
+import { type Image } from '#/modules/home/types/image.type.ts';
+
 export type Accommodation = {
   id: string;
   name: string;
   description: string;
-  images: string[];
+  images: string[] | Image[];
```

Comment:
it is better for this to be just ```Image[]``` because backend does not send array of strings anymlre